examples/BrightnessAndContrast/BrightnessAndContrast.ino

```diff
@@ -3,12 +3,17 @@
 
 // создаем объект для работы с дисплеем
 TroykaTextLCD lcd;
+// по умолчанию в параметры конструктора дисплея зашиты параметры:
+// выбор шины I²C (&Wire), I²C-адрес и пин подсветки
+// TroykaTextLCD lcd(&Wire, 0x3E, 7);
```

Comment:
It`s bad. The backlight pin has a default value (7). If a person uses this pin for other modules, It can crush his program.

Reply:
Create constructor like:
`
TroykaTextLCD::TroykaTextLCD(uint8_t pinBacklight) {
  TroykaTextLCD(&Wire, 0x3e, pinBacklight);
}
`

Reply:
@yurybotov  I think to leave it as it is, so as not to get confused:
1. Create a constructor with default parameters:
TroykaTextLCD lcd;
2. Set parameters manually.
// TroykaTextLCD lcd (& Wire, 0x3E, 7);